### PR TITLE
light theme mode added and exported

### PR DESCRIPTION
**Pull request by AnnaLysiuk**

### CODE REVIEW

File: src/Theme/DefaultDiagramThemeLight.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { DiagramTheme } from "../Models";
+import { DefaultDiagramTheme } from "./DefaultDiagramTheme";
+import { Colors } from "./Colors";
+
+export const DefaultDiagramThemeLight: DiagramTheme = {
+  ...JSON.parse(JSON.stringify(DefaultDiagramTheme)),
```
Comment (hzub) [inline src/Theme/DefaultDiagramThemeLight.ts]: why not `...DefaultDiagramTheme`, are there any issues with this approach?

File: src/Theme/Colors.ts [hunk 2]
```diff
@@ -9,7 +9,9 @@ export const Colors = {
     "#333232",
     "#202020",
     "#1A1919",
-    "#050505"
+    "#050505",
+    "#E5E5E5",
```
Comment (aexol) [inline src/Theme/Colors.ts]: Colors are added from lightest to darkest so you need to chang order

File: src/Theme/Colors.ts [hunk 3]
```diff
@@ -36,7 +38,8 @@ export const Colors = {
     "#B39622",
     "#806B18",
     "#4D400F",
-    "#332B0A"
+    "#332B0A",
+    "#C3A240"
```
Comment (aexol) [inline src/Theme/Colors.ts]: same here , wrong order of colours

**Comment by aexol** [comment]
make ColorsLight object lik e colors

**Pull request closed without merging**